src/Microsoft.AspNet.Http.Abstractions/FragmentString.cs

```diff
@@ -100,7 +100,7 @@ public static FragmentString FromUriComponent(Uri uri)
             string fragmentValue = uri.GetComponents(UriComponents.Fragment, UriFormat.UriEscaped);
             if (!string.IsNullOrEmpty(fragmentValue))
             {
-                fragmentValue = "#" + fragmentValue;
+                fragmentValue = $"#{fragmentValue}";
```

Comment:
I don't think it is: https://github.com/aspnet/HttpAbstractions/pull/411#discussion_r43713247